agent/whatweb_agent.py

```diff
@@ -230,7 +230,11 @@ def _prepare_ip_targets(self, message: msg.Message) -> List[IPTarget]:
         if mask is None:
             network = ipaddress.ip_network(f"{host}")
         else:
-            version = message.data.get("version")
+            try:
+                ip = ipaddress.ip_address(host)
+                version = ip.version
+            except ValueError:
+                raise ValueError(f"Invalid IP address: {host}")
```

Comment:
it would be better to re-raise a custom exception, that way we can catch it outside without the fear of silencing another unrelated `ValueError`